src/Config/ObjectModel/DatabaseType.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config.ObjectModel;
+
+public enum DatabaseType
+{
+    CosmosDB_NoSQL,
+    MySQL,
+    MSSQL,
```

Comment:
nit: I believe we were use all lowercase type names, does the case matter in the json config?

Reply:
No, the introduction of the `EnumMemberJsonEnumConverterFactory` means that it will treat them in a case insensitive manner. This means .NET can be cased per .NET naming conventions but the JSON it won't matter, just use `_` in the right spot.